sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -4,6 +4,8 @@
 
 ```ts
 
+/// <reference lib="esnext.asynciterable" />
```

Comment:
This could be troublesome if a user doesn't have asynciterable support at runtime. Are we using asynciterator-polyfill in this package? Maybe that is where this comes from?

Reply:
Good question - I don't see it as a direct reference but it does look it gets pulled in indirectly from our core packages

```bash
@rush-temp/communication-chat 0.0.0
├─┬ @azure/communication-identity 1.0.0
│ ├─┬ @azure/communication-common 1.1.0
│ │ └─┬ @azure/core-http 2.2.0
│ │   └── @azure/core-asynciterator-polyfill 1.0.0
│ ├─┬ @azure/core-http 1.2.6
│ │ └── @azure/core-asynciterator-polyfill 1.0.0
│ ├─┬ @azure/core-lro 1.0.5
│ │ └─┬ @azure/core-http 1.2.6
│ │   └── @azure/core-asynciterator-polyfill 1.0.0
│ └─┬ @azure/core-paging 1.2.0
│   └── @azure/core-asynciterator-polyfill 1.0.0
```

I also checked the rollup dts file and it looks like that same reference shows up there on main, but not on API Extractor. I feel like that's a good thing though - the dts files aren't tracked in git, and aren't reviewed so such things can go unnoticed. Now that API Extractor surfaces them we can get a better picture of things.

Reply:
FWIW we're supporting node12 and above only and IIRC async iterators were supported in v10 so it should be ok for node customers.